src/goRunTestCodelens.ts

```diff
@@ -139,4 +140,26 @@ export class GoRunTestCodeLensProvider extends GoBaseCodeLensProvider {
 
 		return Promise.all([testPromise, benchmarkPromise]).then(() => codelens);
 	}
+
+	private getDebugConfig(vsConfig: vscode.WorkspaceConfiguration): any {
+		let debugConfig: any = this.defaultDebugConfig;
+		let delveConfig: any = vsConfig.get('delveConfig');
```

Comment:
The same needs to be done in the `resolveDebugConfiguration` method at https://github.com/Microsoft/vscode-go/blob/master/src/goDebugConfiguration.ts. So we should probably move this to the util.ts file

This is what is used when we debug "main" programs without any launch.json files

Reply:
Ok, so I unified the default configurations for both normal and test codelens debug. This should get rid of the confusion caused by this difference, plus being configurable for delve